frontend/src/components/common/Dropdown.tsx

```diff
@@ -1,10 +1,10 @@
 import Css from "./Dropdown.module.css";
 import * as Feather from "react-feather";
 export default function Dropdown(props: {
-    choices: string[];
-    selected: string;
+    choices: (string | number)[];
+    selected: string | number;
     onSelect: (selected: number) => void;
-    emptyPlaceholder: string;
+    emptyPlaceholder: string | number;
```

Comment:
These change are not necessary because you can simply cast everything to string (using `Number.prototype.toString`) before passing them into this component

Reply:
I will revert it back, and use string representation as you recommended.